templates/web/base/report/_item.html

```diff
@@ -46,7 +46,7 @@ <h3 class="item-list__heading">[% problem.title | html %]</h3>
             [% END %]
             [%- IF c.cobrand.moniker != 'fixamingata' %] [%# Default: %]
                 [%- IF problem.days_ago <= 28 %]
-                  [% tprintf(loc('Reported %s days ago'), problem.days_ago) %]
+                  [% tprintf(loc('Reported %s %s ago'), problem.days_ago, nget( loc('week'), loc('weeks'), problem.days_ago ) ) %]
```

Comment:
Please translate full sentences, because it is impossible for some languages to translate things as you have them here :)